aldryn_config.py

```diff
@@ -13,16 +16,37 @@ def clean(self, value):
         class_names = filter(bool, map(lambda x: x.strip(), value.split(',')))
         for class_name in class_names:
             if not CLASS_NAME_FORMAT.match(class_name):
-                raise forms.ValidationError(u'%s is not a proper class name.' % (class_name, ))
-        return u", ".join(class_names)
+                raise forms.ValidationError(
+                    u'{} is not a proper class name.'.format(class_name)
```

Comment:
This will fail on python 2.6, please pass indexes or names. like '{0}'.format(). Otherwise you will get ``ValueError: zero length field name in format``
Though I'm not really sure if we support python 2.6 with this add-on...